test/integration/sky.cc

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/Image.hh>
+
+#include "test_config.h"  // NOLINT(build/include)
+
+#include "ignition/rendering/Camera.hh"
+#include "ignition/rendering/Image.hh"
+#include "ignition/rendering/PixelFormat.hh"
+#include "ignition/rendering/RenderEngine.hh"
+#include "ignition/rendering/RenderingIface.hh"
+#include "ignition/rendering/Scene.hh"
+
+using namespace ignition;
+using namespace rendering;
+
+class SkyTest: public testing::Test,
+               public testing::WithParamInterface<const char *>
+{
+  // Test and verify sky is created
+  public: void Sky(const std::string &_renderEngine);
+};
+
+/////////////////////////////////////////////////
+void SkyTest::Sky(const std::string &_renderEngine)
+{
+  // create and populate scene
+  RenderEngine *engine = rendering::engine(_renderEngine);
+  if (!engine)
+  {
+    igndbg << "Engine '" << _renderEngine
+              << "' is not supported" << std::endl;
```

Comment:
I think these debug messages won't show. Adding this above should do it:

```
  // Documentation inherited
  public: void SetUp() override
  {
    ignition::common::Console::SetVerbosity(4);
  }
```

Reply:
thanks, added. 0823885